ext/node/polyfills/internal_binding/stream_wrap.ts

```diff
@@ -153,6 +153,12 @@ export class LibuvStreamWrap extends HandleWrap {
   readStop(): number {
     this.#reading = false;
 
+    // Unref any reads that are pending until this point.
+    if (this.unref) {
+      this.unref();
+      this.ref();
+    }
```

Comment:
Is this correct that we're unrefing and refing immediately after?

Reply:
yup, this is to unref any ongoing reads until now but not any future reads.